### PR TITLE
Move 'cp-kyma-getting-started' from QA to Production

### PR DESCRIPTION
**Pull request by KevinMuessig**

### CODE REVIEW

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+### You will learn
+  - How to enable the Kyma runtime trial on your SAP Cloud Platform account.
+
+---
+
+[ACCORDION-BEGIN [Step 1: ](Enable the needed Entitlements)]
+
+In case the entitlements for the Kyma runtime are not enabled yet you need to do so in order to enable the Kyma runtime trial.
+
+Click on the **Entitlements** menu item on the left-hand-side.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:23]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-01.png)
+
+Use the search field to search for **Kyma** in order to check if you have the needed entitlements already enabled. If you have a fairly new trial account the entitlement should automatically be added to your account as seen in the screenshot below.
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-02.png)
+
+In case you don't have the entitlement in your account you need to add it now in order to continue. Without the entitlement you won't be able to enable the Kyma runtime for your SAP Cloud Platform account.
+
+Click on **Configure Entitlements**.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:33]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 3]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-02b.png)
+
+Click on **Add Service Plans** to add the Kyma Runtime entitlement to your account.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:37]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 4]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-02c.png)
+
+Search for **Kyma**, check the trial checkbox and click on **Add 1 Service Plan**.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:41]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-02d.png)
+
+> In case you see the message **You have already used all the global account quota for this service plan. To add this service plan, purchase more quota or remove quota from other subaccounts** You already have the Kyma Runtime entitlement used in another subaccount. Look for the subaccount where the Kyma runtime entitlement is assigned too and delete it or use it there.
+
+Click on **Save** to save the newly added service plan.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:47]: Don't say "click on".

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 6]
```diff
@@ -0,0 +1,138 @@
...
+If you want to learn more about entitlements, visit the [Add a New Entitlement to Your Subaccount](https://developers.sap.com/tutorials/cp-cf-entitlements-add.html) tutorial.
+
+Go back to the **Overview** with help of the menu item on the left-hand-side. You should see the **Kyma Environment** tab being displayed.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 2: ](Enable the Kyma Environment)]
+
+Now that you've made sure the Kyma entitlement is existing, you can click on the **Enable Kyma** button to trigger the enablement of the Kyma environment on your SAP Cloud Platform account. This will cause the SAP Cloud Platform to prepare everything for you to create a new cluster at a later point where you can use your Kyma environment on.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:58]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 7]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-03.png)
+
+In the upcoming dialogue enter your cluster name of choice and click **Create**.
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-03b.png)
+
+What happens now is that, in the background, a fresh and new Kubernetes cluster is spinning up where the Kyma runtime and all its components are being installed. This might take a while until it's finished with the setup process.
+
+Kyma itself is a runtime deployed on a Kubernetes cluster and is using all sorts of services to provide you an easy to use environment for your applications and services to run in. In example, the Kyma runtime is using `Istio` to bring a Service Mesh to you.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:68]: "In example"?

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 8]
```diff
@@ -0,0 +1,138 @@
...
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 3: ](Create a new Role Collection for your Kyma User)]
+
+You have your Kyma runtime enabled and a cluster created, the last step is to give your user the needed role to be able to access the Kyma console UI. There are two different roles available for you to choose from `KymaRuntimeNamespaceAdmin` and `KymaRuntimeNamespaceUser`. In this tutorial you will give your user the admin role as it allows you to have full access to all functionality provided by the Kyma runtime trial.
+
+To do so you need to create a Role Collection where you assign the needed role. This will allow you to later on add additional role to it if you're using other services of the SAP Cloud Platform in combination with Kyma runtime.
+
+Click on the **Security** menu item on the left-hand-side and navigate into **Role Collections**. From there click on the **+** icon to create a new collection.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:92]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 9]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-04.png)
+
+In the upcoming dialogue, provide a name for the collection and click on **Create**.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:96]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 10]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-05.png)
+
+Locate your created role collection in the table below and click on its name to open the collection detail screen.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:100]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 11]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-06.png)
+
+In the role collection detail screen, click on **Edit** to add new roles to it.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:104]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 12]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-07.png)
+
+Locate the two-box-icon in order to open up the roles catalogue. The roles catalogue contains all available roles on SAP Cloud Platform including the Kyma runtime roles.
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-08.png)
+
+Search for the role `KymaRuntimeNamespaceAdmin` and add it to your collection by checking the checkbox. Click on **Add** to finish up the process.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:112]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 13]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-09.png)
+
+While still in edit mode we want to assign your user id to the role collection. Locate the **Users** section and enter your email address/ user id into the **ID** field, the UI should give you a nice autocompletion.
+
+Click on **Save**.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:118]: Don't say "click on"

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md [hunk 14]
```diff
@@ -0,0 +1,138 @@
...
+
+![SAP-Cloud-Platform-Trial](kyma-getting-started-10.png)
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Open the Kyma runtime console UI)]
+
+Go back to the overview of your subaccount, if not already done. From there click on **Link to Dashboard**.
```
Comment (MichaelCzcz) [inline tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:127]: Don't say "click on"

**Review by MichaelCzcz — CHANGES_REQUESTED** [review]

**Comment by MichaelCzcz** [comment]
Please sign CLA

File: tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md (hunk 1 above)
Comment (KevinMuessig) [reply tutorials/cp-kyma-getting-started/cp-kyma-getting-started.md:23]: What is it if it's not a click?

**Pull request merged**